mindmeld/cli.py

```diff
@@ -785,6 +792,36 @@ def _get_auto_annotator_config(app_path, overwrite=False, unannotate_all=False):
     return config
 
 
+@shared_cli.command("augment", context_settings=CONTEXT_SETTINGS)
+@click.option(
+    "--app-path",
+    required=True,
+    help="The application's path.",
+)
+@click.option(
+    "--lang",
+    help="Augmentation language.",
+)
+@click.option(
+    "--batch_size",
```

Comment:
not sure we should expose this?

Reply:
Discussing with @vembar , we thought this might be useful for individuals to modify depending on the processing capabilities of their machines?

Reply:
@vijay120 increasing or decreasing batch size will affect latency. So thought it might be useful to expose to the user?

Reply:
Is latency important during augmentation time? Also does the latency vary a lot depending on the batch size? I would prefer more parsimonious configs in the CLI flags to prevent developers from grokking too much and trying the tool out, while using `config.py` to do more advanced changes. However, if we think batch_size is something developers have to worry about when trying out the tool, we can add that.

Reply:
Removed from CLI, can be still modified in the config.